blockchain/common.go

```diff
@@ -24,13 +24,18 @@ var blockchains = []string{
 	ONT,
 	BSC,
 	NEAR,
+	ETH_QAE,
 }
 
 type Params struct {
-	Endpoint   string   `json:"endpoint"`
-	Addresses  []string `json:"addresses"`
-	Topics     []string `json:"eventTopics"`
-	AccountIds []string `json:"accountIds"`
+	Endpoint    string          `json:"endpoint"`
+	Addresses   []string        `json:"addresses"`
+	Topics      []string        `json:"eventTopics"`
+	AccountIds  []string        `json:"accountIds"`
+	Address     string          `json:"address"`
```

Comment:
Do we need to separate `Address` from `Addresses`? This already starts to look like a bucket where we put everything. I would love it if we could separate these params somehow.

Reply:
My plan with this was to slowly migrate `Addresses` over to `Address`, since we in most cases only want to support a single address anyways.
In the future, we can check if `Addresses` is empty, and default to `Address` instead.

Do you think we can improve the approach on this?